### ISSUE TITLE
Not correctly generating releases

### ISSUE THREAD

**Issue by Levi-Armstrong** [tesseract-robotics/tesseract_ros2#80]
Current releases have not been going through the correct release process so the package.xml and changelog has not been updated since version 0.18.X. Who ever is doing the releases for this package please make sure to do this. @marip8

**Comment by marip8** [comment]
Correct. I've been generating releases to trigger the CI build that pushes docker images, since that is currently the only way to trigger the docker push stage of CI. These releases were for tags that were added in the history, so I was unable to go through the normal `catkin_prepare_release` process (for some of them at least).

Going forward, we should apply the `catkin_prepare_release` steps, at least for changes in minor versions. We'll just have to be cognizant of doing this **before** merging minor-version-breaking changes.

With regard to the releases I've been making recently, this is clearly not the best way of trigger the generation of new docker images for specific tags. What I would like to do is use the `workflow_dispatch` to trigger the CI and change the rule for the push step to allow the image to be pushed if the branch is a tag. I have yet to figure out how to do this successfully in Github Actions though
